interface-definitions/system_frr.xml.in

```diff
@@ -35,6 +35,22 @@
               <valueless/>
             </properties>
           </leafNode>
+          <leafNode name="profile">
+            <properties>
+              <help>FRR profile</help>
+              <completionHelp>
+                <list>traditional datacenter</list>
+              </completionHelp>
+              <valueHelp>
+                <format>traditional</format>
+                <description>Reflects defaults adhering mostly to IETF standards or common practices in wide-area internet routing</description>
+              </valueHelp>
+              <valueHelp>
+                <format>datacenter</format>
+                <description>Reflects a single administrative domain with intradomain links using aggressive timers</description>
+              </valueHelp>
```

Comment:
A `regex` constraint is missing here. Without the constraint one can fantasize any profile.

```
cpo@LR1.wue3# set system frr profile trad-foo-datacenter
[edit]
```